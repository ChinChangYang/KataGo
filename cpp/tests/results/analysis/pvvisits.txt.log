: Running with following config:
cudaUseFP16 = false
cudaUseNHWC = false
forDeterministicTesting = true
logAllRequests = true
logAllResponses = true
logFile = tests/results/analysis/pvvisits.txt.log
logSearchInfo = true
logTimeStamp = false
maxPlayouts = 10000
maxVisits = 100
nnCacheSizePowerOfTwo = 23
nnMaxBatchSize = 64
nnMutexPoolSizePowerOfTwo = 17
nnRandSeed = analysisTest
nnRandomize = false
numAnalysisThreads = 1
numSearchThreads = 1
openclUseFP16 = false
reportAnalysisWinratesAs = BLACK
rootSymmetryPruning = false
trtUseFP16 = false

: Analysis Engine starting...
: KataGo v1.16.0
: nnRandSeed0 = analysisTest
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Initializing neural net buffer to be size 19 * 19 allowing smaller boards
: Cuda backend thread 0: Found GPU NVIDIA RTX A5000 memory 25425608704 compute capability major 8 minor 6
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded config configs/analysis_example.cfg and/or command-line and query overrides
: Loaded model tests/models/g170-b6c96-s175395328-d26788732.bin.gz
: Config override: cudaUseFP16 = false
: Config override: cudaUseNHWC = false
: Config override: forDeterministicTesting = true
: Config override: logAllRequests = true
: Config override: logAllResponses = true
: Config override: logDir = 
: Config override: logFile = tests/results/analysis/pvvisits.txt.log
: Config override: logSearchInfo = true
: Config override: logTimeStamp = false
: Config override: maxPlayouts = 10000
: Config override: maxVisits = 100
: Config override: nnRandSeed = analysisTest
: Config override: nnRandomize = false
: Config override: numAnalysisThreads = 1
: Config override: numSearchThreadsPerAnalysisThread = 1
: Config override: openclUseFP16 = false
: Config override: rootSymmetryPruning = false
: Config override: trtUseFP16 = false
: Analyzing up to 1 positions at a time in parallel
: Started, ready to begin handling requests
: Request: {"id":"pvvisittest","initialStones":[],"moves":[["B","D4"],["W","D5"],["B","C5"],["W","E4"],["B","D3"],["W","C6"],["B","E5"],["W","D6"]],"rules":"tromp-taylor","komi":9,"boardXSize":7,"boardYSize":7,"overrideSettings":{"maxVisits":1000},"includePVVisits":true}
: Request: {"id":"pvvisittest","initialStones":[],"moves":[["B","D4"],["W","D5"],["B","C5"],["W","E4"],["B","D3"],["W","E5"],["B","C6"],["W","E3"],["B","E2"],["W","F2"],["B","D2"],["W","C4"],["B","D6"],["W","B4"]],"rules":"chinese","komi":9,"boardXSize":7,"boardYSize":7,"overrideSettings":{"maxVisits":1000},"includePVVisits":true}
: MoveNum: 8 HASH: 68923BD112145D32565C85CE0EF9CB35
   A B C D E F G
 7 . . . . . . .
 6 . . O1O3. . .
 5 . . X O X2. .
 4 . . . X O . .
 3 . . . X . . .
 2 . . . . . . .
 1 . . . . . . .

koPOSITIONALscoreAREAtaxNONEsui1komi9
Root visits: 1000
New playouts: 1000
NN rows: 872
NN batches: 872
NN avg batch size: 1
PV: E3 F4 F5 F6 F3 C4 B5 B4 B6 B7 B3 A5 C3 A6 E6
Tree:
: T   5.29c W  13.09c S  -4.36c ( +0.4 L  +0.4) N    1000  --  E3 F4 F5 F6 F3 C4 B5
---Black(^)---
E3  : T  18.54c W  23.74c S  -4.09c ( +0.9 L  +0.9) LCB    6.48c P  2.45% WF 494.4 PSV     440 N     449  --  E3 F4 F5 F6 F3 C4 B5 B4
F4  : T -10.72c W  -2.27c S  -5.69c ( -0.2 L  -0.2) LCB  -31.68c P 40.28% WF 148.9 PSV      84 N     170  --  F4 C4 C3 B5 B4 F5 E3 E6
E6  : T -24.15c W -12.02c S  -5.04c ( -0.9 L  -1.1) LCB  -46.25c P 38.55% WF  71.6 PSV      55 N      91  --  E6 E3 E2 F2 B5 B6 D2 F5
F5  : T  -2.23c W   8.82c S  -4.25c ( +0.2 L  +0.1) LCB  -19.84c P 17.71% WF 272.5 PSV      52 N     288  --  F5 B5 C4 B4 B3 E3 D2 E2
B5  : T -85.40c W -71.58c S -13.82c ( -4.9 L  -4.3) LCB -435.40c P  0.79% WF   0.8 PSV       0 N       1  --  B5

: Response: {"id":"pvvisittest","isDuringSearch":false,"moveInfos":[{"edgeVisits":449,"edgeWeight":440.241926,"lcb":0.575651855,"move":"E3","order":0,"playSelectionValue":440.241926,"prior":0.0245141629,"pv":["E3","F4","F5","F6","F3","C4","B5","B4","B6","B7","B3","A5","C3","A6","E6"],"pvEdgeVisits":[449,262,254,206,189,93,74,61,44,15,9,6,4,3,1],"pvVisits":[449,262,254,206,189,93,74,61,44,15,9,6,4,3,1],"scoreLead":0.889067096,"scoreMean":0.889067096,"scoreSelfplay":0.942757309,"scoreStdev":7.02944441,"utility":0.185374803,"utilityLcb":0.064771482,"visits":449,"weight":440.241926,"winrate":0.61872447},{"edgeVisits":170,"edgeWeight":169.588382,"lcb":0.413816464,"move":"F4","order":1,"playSelectionValue":84.0,"prior":0.402848393,"pv":["F4","C4","C3","B5","B4","F5","E3","E6","C5","E4","B6","C4","E5"],"pvEdgeVisits":[170,132,111,106,77,34,20,17,6,5,3,2,1],"pvVisits":[170,132,111,106,77,34,20,17,6,5,3,2,1],"scoreLead":-0.225639281,"scoreMean":-0.225639281,"scoreSelfplay":-0.207346374,"scoreStdev":9.50004228,"utility":-0.107215932,"utilityLcb":-0.316811232,"visits":170,"weight":169.588382,"winrate":0.488671929},{"edgeVisits":91,"edgeWeight":90.912394,"lcb":0.360962268,"move":"E6","order":2,"playSelectionValue":55.0,"prior":0.385524601,"pv":["E6","E3","E2","F2","B5","B6","D2","F5","F6","F4"],"pvEdgeVisits":[91,60,31,28,5,3,4,3,2,1],"pvVisits":[91,60,31,28,5,7,4,3,2,1],"scoreLead":-1.10799779,"scoreMean":-1.10799779,"scoreSelfplay":-0.928618196,"scoreStdev":15.7591917,"utility":-0.241508506,"utilityLcb":-0.462474778,"visits":91,"weight":90.912394,"winrate":0.439878793},{"edgeVisits":288,"edgeWeight":286.406582,"lcb":0.481216562,"move":"F5","order":3,"playSelectionValue":52.0,"prior":0.17705363,"pv":["F5","B5","C4","B4","B3","E3","D2","E2","F4","B2","A4","A5","B6"],"pvEdgeVisits":[288,257,157,117,91,80,70,46,21,17,6,4,1],"pvVisits":[288,257,157,117,91,80,70,46,21,17,6,4,1],"scoreLead":0.143852925,"scoreMean":0.143852925,"scoreSelfplay":0.200830286,"scoreStdev":11.6686185,"utility":-0.0222511162,"utilityLcb":-0.19837005,"visits":288,"weight":286.406582,"winrate":0.544116181},{"edgeVisits":1,"edgeWeight":1.0,"lcb":-1.10789722,"move":"B5","order":4,"playSelectionValue":0.0,"prior":0.00792252645,"pv":["B5"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-4.3143425,"scoreMean":-4.3143425,"scoreSelfplay":-4.92409325,"scoreStdev":10.3025147,"utility":-0.854036611,"utilityLcb":-4.35403663,"visits":1,"weight":1.0,"winrate":0.142102785}],"rootInfo":{"currentPlayer":"B","rawLead":2.50582242,"rawNoResultProb":0.0,"rawScoreSelfplay":3.91776896,"rawScoreSelfplayStdev":9.33004979,"rawStScoreError":-1.0,"rawStWrError":-0.5,"rawVarTimeLeft":-1.0,"rawWinrate":0.799173474,"scoreLead":0.50850346,"scoreSelfplay":0.570843417,"scoreStdev":8.92973607,"symHash":"10155F672C0B5682E08EB339027AD849","thisHash":"E2B811FEE059E65F93697CB1DE35B543","utility":0.0929851185,"visits":1000,"weight":989.1492838563422,"winrate":0.580036577},"turnNumber":8}
: MoveNum: 14 HASH: B56076090A467FAA5A715507E32E1149
   A B C D E F G
 7 . . . . . . .
 6 . . X X2. . .
 5 . . X O O . .
 4 . O3O1X O . .
 3 . . . X O . .
 2 . . . X X O .
 1 . . . . . . .

koSIMPLEscoreAREAtaxNONEsui0whbNfpok1komi9
Root visits: 1000
New playouts: 1000
NN rows: 1763
NN batches: 1763
NN avg batch size: 1
PV: C3 E6 F1 B5 B6 D7 A6 F3 C7 E7 A4 G2 E1 G1 B3 pass A5 pass pass
Tree:
: T  29.21c W  30.56c S  -2.11c ( +1.1 L  +1.0) N    1000  --  C3 E6 F1 B5 B6 D7 A6
---Black(^)---
C3  : T  35.51c W  36.11c S  -1.67c ( +1.4 L  +1.3) LCB   28.11c P  9.61% WF 832.2 PSV     787 N     793  --  C3 E6 F1 B5 B6 D7 A6 F3
F1  : T   6.95c W   9.64c S  -1.95c ( +0.7 L  +0.7) LCB  -14.77c P 26.21% WF  80.2 PSV      48 N      98  --  F1 E6 B5 B2 C2 B3 F3 D7
B2  : T   4.42c W   8.33c S  -4.08c ( +0.1 L  +0.2) LCB  -33.88c P 19.59% WF  60.8 PSV      33 N      75  --  B2 E6 B5 D7 C7 E7 F1 A6
E6  : T -70.32c W -48.50c S -12.16c ( -6.6 L  -6.9) LCB -125.53c P 20.52% WF   7.3 PSV      10 N      13  --  E6 C3 C2 B2 B1
B5  : T -56.58c W -49.01c S -10.78c ( -5.9 L  -5.7) LCB  -88.75c P 10.33% WF   5.1 PSV       6 N       8  --  B5 B2 C2 B3 F1
B3  : T -88.35c W -51.04c S -16.58c (-10.7 L  -9.1) LCB -146.49c P 12.47% WF   4.4 PSV       5 N       8  --  B3 C3 C2 B2 F1
F3  : T -57.31c W -46.97c S -10.34c ( -4.7 L  -4.4) LCB -407.31c P  0.93% WF   0.8 PSV       0 N       1  --  F3
G2  : T -68.87c W -56.05c S -12.82c ( -6.5 L  -5.7) LCB -418.87c P  0.10% WF   0.7 PSV       0 N       1  --  G2
F6  : T -99.44c W -79.76c S -19.68c (-13.9 L -12.0) LCB -449.44c P  0.08% WF   0.7 PSV       0 N       1  --  F6
C2  : T -62.23c W -52.64c S  -9.59c ( -3.8 L  -2.7) LCB -412.23c P  0.05% WF   0.7 PSV       0 N       1  --  C2

: Response: {"id":"pvvisittest","isDuringSearch":false,"moveInfos":[{"edgeVisits":793,"edgeWeight":787.033095,"lcb":0.654133537,"move":"C3","order":0,"playSelectionValue":787.033095,"prior":0.0960594565,"pv":["C3","E6","F1","B5","B6","D7","A6","F3","C7","E7","A4","G2","E1","G1","B3","pass"],"pvEdgeVisits":[793,509,488,375,373,107,89,73,68,57,18,13,10,8,6,5],"pvVisits":[793,509,488,375,373,107,89,73,68,57,18,13,10,8,6,5],"scoreLead":1.32268977,"scoreMean":1.32268977,"scoreSelfplay":1.39945705,"scoreStdev":9.88871386,"utility":0.355102562,"utilityLcb":0.281111844,"visits":793,"weight":787.033095,"winrate":0.680558793},{"edgeVisits":98,"edgeWeight":97.8678664,"lcb":0.470641396,"move":"F1","order":1,"playSelectionValue":48.0,"prior":0.262068182,"pv":["F1","E6","B5","B2","C2","B3","F3","D7","C7","E7"],"pvEdgeVisits":[98,57,52,46,22,15,11,4,2,1],"pvVisits":[98,57,52,46,22,15,11,4,2,1],"scoreLead":0.705965628,"scoreMean":0.705965628,"scoreSelfplay":0.745707458,"scoreStdev":17.3115702,"utility":0.0695191987,"utilityLcb":-0.147711052,"visits":98,"weight":97.8678664,"winrate":0.548223628},{"edgeVisits":75,"edgeWeight":75.0,"lcb":0.4048703,"move":"B2","order":2,"playSelectionValue":33.0,"prior":0.195859268,"pv":["B2","E6","B5","D7","C7","E7","F1","A6","A4","A5"],"pvEdgeVisits":[75,65,60,32,27,18,12,5,2,1],"pvVisits":[75,65,60,32,27,18,12,5,2,1],"scoreLead":0.198402337,"scoreMean":0.198402337,"scoreSelfplay":0.146948438,"scoreStdev":10.1657782,"utility":0.044204957,"utilityLcb":-0.338783365,"visits":75,"weight":75.0,"winrate":0.541651843},{"edgeVisits":13,"edgeWeight":13.0,"lcb":0.0603277192,"move":"E6","order":3,"playSelectionValue":10.0,"prior":0.205167174,"pv":["E6","C3","C2","B2","B1"],"pvEdgeVisits":[13,9,3,2,1],"pvVisits":[13,9,3,2,1],"scoreLead":-6.85521027,"scoreMean":-6.85521027,"scoreSelfplay":-6.56721905,"scoreStdev":16.1268306,"utility":-0.703174059,"utilityLcb":-1.25528499,"visits":13,"weight":13.0,"winrate":0.257510195},{"edgeVisits":8,"edgeWeight":8.0,"lcb":0.140059273,"move":"B5","order":4,"playSelectionValue":6.0,"prior":0.103304945,"pv":["B5","B2","C2","B3","F1"],"pvEdgeVisits":[8,7,3,2,1],"pvVisits":[8,7,3,2,1],"scoreLead":-5.72120836,"scoreMean":-5.72120836,"scoreSelfplay":-5.94338389,"scoreStdev":17.3595111,"utility":-0.565773393,"utilityLcb":-0.887469038,"visits":8,"weight":8.0,"winrate":0.254950574},{"edgeVisits":8,"edgeWeight":8.0,"lcb":0.0371825841,"move":"B3","order":5,"playSelectionValue":5.0,"prior":0.124727048,"pv":["B3","C3","C2","B2","F1"],"pvEdgeVisits":[8,7,3,2,1],"pvVisits":[8,7,3,2,1],"scoreLead":-9.08220855,"scoreMean":-9.08220855,"scoreSelfplay":-10.6990281,"scoreStdev":17.1136089,"utility":-0.883516689,"utilityLcb":-1.46488584,"visits":8,"weight":8.0,"winrate":0.244814422},{"edgeVisits":1,"edgeWeight":1.0,"lcb":-0.98484335,"move":"F3","order":6,"playSelectionValue":0.0,"prior":0.0093464097,"pv":["F3"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-4.40652609,"scoreMean":-4.40652609,"scoreSelfplay":-4.69277668,"scoreStdev":14.6459916,"utility":-0.573112936,"utilityLcb":-4.07311295,"visits":1,"weight":1.0,"winrate":0.265156657},{"edgeVisits":1,"edgeWeight":1.0,"lcb":-1.03024033,"move":"G2","order":7,"playSelectionValue":0.0,"prior":0.000979979755,"pv":["G2"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-5.69929218,"scoreMean":-5.69929218,"scoreSelfplay":-6.5362916,"scoreStdev":14.8008346,"utility":-0.688712086,"utilityLcb":-4.1887121,"visits":1,"weight":1.0,"winrate":0.219759673},{"edgeVisits":1,"edgeWeight":1.0,"lcb":-1.14881623,"move":"F6","order":8,"playSelectionValue":0.0,"prior":0.000808661047,"pv":["F6"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-12.0131226,"scoreMean":-12.0131226,"scoreSelfplay":-13.8879013,"scoreStdev":17.542444,"utility":-0.994437121,"utilityLcb":-4.49443714,"visits":1,"weight":1.0,"winrate":0.101183776},{"edgeVisits":1,"edgeWeight":1.0,"lcb":-1.01320066,"move":"C2","order":9,"playSelectionValue":0.0,"prior":0.000484645163,"pv":["C2"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-2.74764657,"scoreMean":-2.74764657,"scoreSelfplay":-3.83684063,"scoreStdev":13.5833309,"utility":-0.622273407,"utilityLcb":-4.12227342,"visits":1,"weight":1.0,"winrate":0.236799344}],"rootInfo":{"currentPlayer":"B","rawLead":1.90136731,"rawNoResultProb":8.25764291e-05,"rawScoreSelfplay":3.3719821,"rawScoreSelfplayStdev":12.2536581,"rawStScoreError":-1.0,"rawStWrError":-0.5,"rawVarTimeLeft":-1.0,"rawWinrate":0.689348087,"scoreLead":1.05057355,"scoreSelfplay":1.11299975,"scoreStdev":10.7300678,"symHash":"0CAA03496D1088226717248B65E75E44","thisHash":"3F4A5C26F80BC4C79F44AC7833E26F3F","utility":0.303166429,"visits":1000,"weight":993.8641158660905,"winrate":0.658211214},"turnNumber":14}
: tests/models/g170-b6c96-s175395328-d26788732.bin.gz
: NN rows: 1763
: NN batches: 1763
: NN avg batch size: 1
: GPU -1 finishing, processed 1763 rows 1763 batches
: All cleaned up, quitting
