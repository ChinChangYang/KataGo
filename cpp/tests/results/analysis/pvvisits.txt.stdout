{"id":"pvvisittest","isDuringSearch":false,"moveInfos":[{"lcb":0.5789529,"move":"E3","order":0,"prior":0.0245399754,"pv":["E3","F4","F5","F6","F3","C4","B5","B4","B6","B7","B3","A5","C3","A6","E6"],"pvEdgeVisits":[446,259,251,203,186,90,71,58,43,14,9,6,4,3,1],"pvVisits":[446,259,251,203,186,90,71,58,43,14,9,6,4,3,1],"scoreLead":0.917900076,"scoreMean":0.917900076,"scoreSelfplay":0.971999527,"scoreStdev":7.03699335,"utility":0.194186928,"utilityLcb":0.0743009118,"visits":446,"weight":438.5133035193674,"winrate":0.621769335},{"lcb":0.413677997,"move":"F4","order":1,"prior":0.402511716,"pv":["F4","C4","C3","B5","B4","F5","E3","E6","C5","E4","B6","C4","E5"],"pvEdgeVisits":[170,132,111,106,77,34,20,17,6,5,3,2,1],"pvVisits":[170,132,111,106,77,34,20,17,6,5,3,2,1],"scoreLead":-0.227108723,"scoreMean":-0.227108723,"scoreSelfplay":-0.208944132,"scoreStdev":9.50046975,"utility":-0.107508899,"utilityLcb":-0.317119866,"visits":170,"weight":169.58783820443773,"winrate":0.488539056},{"lcb":0.360842191,"move":"E6","order":2,"prior":0.385828137,"pv":["E6","E3","E2","F2","B5","B6","D2","F5","F6","F4"],"pvEdgeVisits":[91,60,31,28,5,3,4,3,2,1],"pvVisits":[91,60,31,28,5,7,4,3,2,1],"scoreLead":-1.1098043,"scoreMean":-1.1098043,"scoreSelfplay":-0.93034051,"scoreStdev":15.758466,"utility":-0.241694192,"utilityLcb":-0.462759496,"visits":91,"weight":90.91348913680861,"winrate":0.439794085},{"lcb":0.480644789,"move":"F5","order":3,"prior":0.177052975,"pv":["F5","B5","C4","B4","B3","E3","D2","E2","F4","B2","A4","A5","B6"],"pvEdgeVisits":[291,260,160,120,94,83,72,48,21,17,6,4,1],"pvVisits":[291,260,160,120,94,83,72,48,21,17,6,4,1],"scoreLead":0.112933169,"scoreMean":0.112933169,"scoreSelfplay":0.16619797,"scoreStdev":11.7798398,"utility":-0.0217917158,"utilityLcb":-0.196302665,"visits":291,"weight":289.4088709181914,"winrate":0.542970128},{"lcb":-1.10816444,"move":"B5","order":4,"prior":0.00792988669,"pv":["B5"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-4.3214879,"scoreMean":-4.3214879,"scoreSelfplay":-4.93037653,"scoreStdev":10.3015217,"utility":-0.854692074,"utilityLcb":-4.35469209,"visits":1,"weight":1.0,"winrate":0.14183557}],"rootInfo":{"currentPlayer":"B","rawStScoreError":-1.0,"rawStWrError":-0.5,"rawVarTimeLeft":-1.0,"scoreLead":0.53215593,"scoreSelfplay":0.594543809,"scoreStdev":8.92279766,"symHash":"10155F672C0B5682E08EB339027AD849","thisHash":"E2B811FEE059E65F93697CB1DE35B543","utility":0.100765884,"visits":1000,"weight":990.4235017788051,"winrate":0.582729439},"turnNumber":8}
{"id":"pvvisittest","isDuringSearch":false,"moveInfos":[{"lcb":0.633586722,"move":"C3","order":0,"prior":0.0961157158,"pv":["C3","E6","F1","B5","B6","D7","A6","F3","C7","E7","A4","G2","E1","G1","B3","pass"],"pvEdgeVisits":[790,615,590,485,482,236,116,76,69,58,20,15,12,10,6,5],"pvVisits":[790,615,590,485,482,236,116,76,69,58,20,15,12,10,6,5],"scoreLead":0.843264886,"scoreMean":0.843264886,"scoreSelfplay":0.945023316,"scoreStdev":8.44466621,"utility":0.284095303,"utilityLcb":0.210938274,"visits":790,"weight":785.5317603225445,"winrate":0.659714232},{"lcb":0.470650158,"move":"F1","order":1,"prior":0.262209266,"pv":["F1","E6","B5","B2","C2","B3","F3","D7","C7","E7"],"pvEdgeVisits":[98,57,52,46,22,15,11,4,2,1],"pvVisits":[98,57,52,46,22,15,11,4,2,1],"scoreLead":0.706438582,"scoreMean":0.706438582,"scoreSelfplay":0.746484308,"scoreStdev":17.3118356,"utility":0.0697779074,"utilityLcb":-0.14754472,"visits":98,"weight":97.86776775296622,"winrate":0.548265382},{"lcb":0.404915015,"move":"B2","order":2,"prior":0.196017057,"pv":["B2","E6","B5","D7","C7","E7","F1","A6","A4","A5"],"pvEdgeVisits":[75,65,60,32,27,18,12,5,2,1],"pvVisits":[75,65,60,32,27,18,12,5,2,1],"scoreLead":0.197931802,"scoreMean":0.197931802,"scoreSelfplay":0.146456322,"scoreStdev":10.1663136,"utility":0.0443648948,"utilityLcb":-0.338441964,"visits":75,"weight":75.0,"winrate":0.54163175},{"lcb":0.0858497439,"move":"E6","order":3,"prior":0.204878151,"pv":["E6","C3","C2","B2","F1","F3"],"pvEdgeVisits":[15,11,5,4,2,1],"pvVisits":[15,11,5,4,2,1],"scoreLead":-6.74523864,"scoreMean":-6.74523864,"scoreSelfplay":-6.38841366,"scoreStdev":16.106698,"utility":-0.664532149,"utilityLcb":-1.14961945,"visits":15,"weight":15.0,"winrate":0.259095207},{"lcb":0.184049043,"move":"B5","order":4,"prior":0.103183784,"pv":["B5","B2","C2","B3","F1","F3"],"pvEdgeVisits":[9,8,4,3,2,1],"pvVisits":[9,8,4,3,2,1],"scoreLead":-5.92479361,"scoreMean":-5.92479361,"scoreSelfplay":-6.148905,"scoreStdev":17.6895101,"utility":-0.604332851,"utilityLcb":-0.783440354,"visits":9,"weight":9.0,"winrate":0.248016008},{"lcb":0.0372259993,"move":"B3","order":5,"prior":0.124776646,"pv":["B3","C3","C2","B2","F1"],"pvEdgeVisits":[8,7,3,2,1],"pvVisits":[8,7,3,2,1],"scoreLead":-9.08064565,"scoreMean":-9.08064565,"scoreSelfplay":-10.6975404,"scoreStdev":17.1134078,"utility":-0.883436064,"utilityLcb":-1.46491427,"visits":8,"weight":8.0,"winrate":0.244896789},{"lcb":-0.984912968,"move":"F3","order":6,"prior":0.0093543632,"pv":["F3"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-4.40707541,"scoreMean":-4.40707541,"scoreSelfplay":-4.69441652,"scoreStdev":14.6451128,"utility":-0.573304598,"utilityLcb":-4.07330462,"visits":1,"weight":1.0,"winrate":0.265087038},{"lcb":-1.03020947,"move":"G2","order":7,"prior":0.00097749487,"pv":["G2"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-5.69696856,"scoreMean":-5.69696856,"scoreSelfplay":-6.53477001,"scoreStdev":14.8005703,"utility":-0.688654861,"utilityLcb":-4.18865488,"visits":1,"weight":1.0,"winrate":0.219790541},{"lcb":-1.14897608,"move":"F6","order":8,"prior":0.000809429854,"pv":["F6"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-12.0237207,"scoreMean":-12.0237207,"scoreSelfplay":-13.9013166,"scoreStdev":17.5407113,"utility":-0.99491842,"utilityLcb":-4.49491844,"visits":1,"weight":1.0,"winrate":0.101023931},{"lcb":-1.01333117,"move":"C2","order":9,"prior":0.00048529284,"pv":["C2"],"pvEdgeVisits":[1],"pvVisits":[1],"scoreLead":-2.75126529,"scoreMean":-2.75126529,"scoreSelfplay":-3.84084296,"scoreStdev":13.5803758,"utility":-0.622635388,"utilityLcb":-4.12263541,"visits":1,"weight":1.0,"winrate":0.23666884}],"rootInfo":{"currentPlayer":"B","rawStScoreError":-1.0,"rawStWrError":-0.5,"rawVarTimeLeft":-1.0,"scoreLead":0.615177801,"scoreSelfplay":0.698658298,"scoreStdev":9.76649778,"symHash":"0CAA03496D1088226717248B65E75E44","thisHash":"3F4A5C26F80BC4C79F44AC7833E26F3F","utility":0.234771789,"visits":1000,"weight":995.3654609835285,"winrate":0.636833763},"turnNumber":14}
